src/bin/hexyl.rs

```diff
@@ -104,24 +104,46 @@ fn run() -> Result<(), AnyhowError> {
                 .takes_value(true)
                 .value_name("WHEN")
                 .possible_values(&["always", "auto", "never"])
+                .default_value_if("plain", None, "never")
                 .default_value("always")
                 .help(
                     "When to use colors. The auto-mode only displays colors if the output \
                      goes to an interactive terminal",
                 ),
         )
+        .arg(
+            Arg::with_name("plain")
+                .short("p")
+                .long("plain")
+                .help("Display output with --no-chars, --no-position, and --border=none."),
+        )
         .arg(
             Arg::with_name("border")
                 .long("border")
                 .takes_value(true)
                 .value_name("STYLE")
                 .possible_values(&["unicode", "ascii", "none"])
+                .default_value_if("plain", None, "none")
                 .default_value("unicode")
                 .help(
                     "Whether to draw a border with Unicode characters, ASCII characters, \
                     or none at all",
                 ),
         )
+        .arg(
+            Arg::with_name("no_chars")
+                .short("C")
+                .long("no-chars")
```

Comment:
Is there any precedent for this option name in other hex viewers? Or for that terminology ("characters"/"chars")? If not, what would be alternatives?

I usually prefer the long options the be self-descriptive (and longer, if necessary). Something like `--no-character-panel`. Especially if we have shell completions and additional short options.

Reply:
I largely referred to variable naming withing the code itself, in hindsight that might not be the best source.
I like `--no-character-panel` though I would propose that the suffix table be used: `--no-character-table` mainly because  `_table` is already used in the codebase:
https://github.com/sharkdp/hexyl/blob/a23645ddfdc1d26a64f2d6d6894ac577058df134/src/lib.rs#L151-L152

Reply:
Hm. I don't like the "table" suffix. Neither in the code, nor in the CLI. If at all, it's a single column of the table. I suggest we either use:
```
--no-position
--no-characters
```
or
```
--no-position-panel
--no-character-panel
```

I'm fine with both alternatives.